lib/matplotlib/figure.py

```diff
@@ -1501,8 +1501,7 @@ def align_labels(self, axs=None):
 
     def add_gridspec(self, nrows=1, ncols=1, **kwargs):
         """
-        Return a `.GridSpec` that has this figure as a parent.  This allows
-        complex layout of Axes in the figure.
+        Create a low-level `.GridSpec` that has this figure as a parent.
```

Comment:
```suggestion
        Create a `.GridSpec` that has this figure as a parent.
        
        This is a low-level API, allowing you to create a gridspec and
        subsequently add subplots based on the gridspec. Most users do
        not need that freedom and should use the higher-level methods
        `~.Figure.subplots` or `~.Figure.subplot_mosaic`.
```

Reply:
This is fine - but the goal here was to indicate in the slug that this is something most folks don't need to click through to.

Reply:
Is "Low-level API for creating a `.GridSpec` that has this figure as a parent." scary enough so that people won't want to know what this is? 😁

Reply:
Done